systems/registry/docker-compose.yml

```diff
@@ -52,6 +52,9 @@ services:
    - NETWORK_SERVICE_PORT=9090
    - PUSHGATEWAY=http://${LOCAL_HOST_IP}:9099
    - ORGHOST=http://${LOCAL_HOST_IP}:8060
+   - COUNTRY=Democratic Democratic Republic of the Congo
+   - CURRENCY=USD
+   - LANGUAGE=EN
```

Comment:
We should use some code standards for country, currency and language to be consistent.

**Some suggestion**
For country we can use:
https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3#Officially_assigned_code_elements

For currency we can use:
https://en.wikipedia.org/wiki/ISO_4217#Active_codes

For language code we can use:
https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes

Reply:
We will incorporate checks at the BFF level to invoke APIs for validating country, currency, and language, thereby enforcing the standards for these fields.